app/Http/Controllers/DuesTransactionController.php

```diff
@@ -120,8 +120,8 @@ public function store(Request $request)
         $existingTransaction = DuesTransaction::where('dues_package_id', $request->input('dues_package_id'))
             ->where('user_id', $request->input('user_id'))->first();
         if ($existingTransaction) {
-            return response()->json(['status' => 'error',
-                'message' => 'There is already a pending Dues Transaction for this user', ], 400);
+            $existingTransaction->touch(); //Update the updated_date on the transaction so it takes precedence
```

Comment:
@RossIV you're okay with this solution for keeping track of the most recently selected transaction?